packages/datatrak-web/src/views/OfflinePage.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button as UIButton } from '@tupaia/ui-components';
+import { Link as RouterLink } from 'react-router-dom';
+import { Typography } from '@material-ui/core';
+import { HEADER_HEIGHT } from '../constants';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+`;
+
+const HeaderContainer = styled.div`
+  background: ${({ theme }) => theme.palette.background.paper};
+`;
+
+const HeaderInner = styled.div`
+  display: flex;
+  align-items: center;
+  block-size: ${HEADER_HEIGHT};
+  padding: 0 1.5rem;
+  margin: 0 auto;
```

Comment:
If I’m not mistaken, `margin-block` is already `0`. Best to set only what we need to override to clean up the cascade for debugging:

```suggestion
  margin-inline: auto;
```